### PR TITLE
Updated documentation for user endpoints

### LINKED ISSUE #9: API + DB model - USERS endpoint
Create endpoints for Users resource:

**USER (Super Admin, Admin, Receptionist, Guest):**
- [ ]      [GET] current user
- [ ]      [GET] users
- [ ]      [PUT] current user (change his user type, user info)
- [ ]      [POST] New company user, available only to Super Admins
- [ ]      [DELETE] User account, available only to Super Admins
- [ ] - **Update documentation on these endpoints!**

**Feature Name:**
Ability to add/remove Receptionists.

- **User access:**
  - Admin
  - Super Admin
- **User Stories:**

  - As a Super Admin/Admin, I want to be able to create a new Receptionist user so that they can serve hotel guests.
  - As a Super Admin, I want to be able to remove a Receptionist if their role changes in the company.

- **What internal APIs will you build to implement this feature?**
  - [POST] New user type
  - [DELETE] User type

### PR DESCRIPTION
**Pull request by DianaIgnatescu**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #9

## Type of change

Please add a relevant label.

## Change status

- [x] Complete, but not tested (may need new tests)

# How Has This Been Tested?

Tested with Postman to check for correct responses

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [ ] I have commented my code, particularly in hard-to-understand areas

- [x] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

### CODE REVIEW

**Review by Pav0l — CHANGES_REQUESTED** [review]
The documentation should go to a folder created by Connor in #28

File: routes/endpoint-docs/userRoutes.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+ *     }
+ * @apiErrorExample Error-Response: User not found
+ *    HTTP/1.1 404 Bad Request
+ *    {
+ *      "message": "The user could not be found"
+ *    }
+ */
+
+/**
+ * [POST] Creates a new user
```
Comment (ConConRob) [inline routes/endpoint-docs/userRoutes.js]: This won't go into docs. Maybe forgot to delete it?

**Review by ConConRob — APPROVED** [review]
The user docs file will work file although a old comment was left in it

File: routes/endpoint-docs/userRoutes.js (hunk 1 above)
Comment (DianaIgnatescu) [reply routes/endpoint-docs/userRoutes.js]: Thanks for the heads up! I removed it now.

**Review by Pav0l — APPROVED** [review]

**Pull request merged**